locales/en-US/faq.properties

```diff
@@ -2,38 +2,56 @@
 faq_item_2_header=How does Mozilla ensure my personal information is protected?
 faq_item_2_paragraph_a=We care about the privacy of our donors and take our <a href="https://www.mozilla.org/privacy/principles/">Privacy Principles</a> seriously. Mozilla believes individuals should be in control of their data, and maintains multi-layered security controls and practices. To learn more, read the <a href="https://www.mozilla.org/privacy/">Mozilla Privacy Policy</a>.
 faq_item_2_paragraph_b=In certain instances, when you donate to Mozilla via a third party like <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">PayPal</a> or <a href="https://stripe.com/privacy/">Stripe</a>, those organizations also collect user data.
+faq_item_2_header_thunderbird=I want to donate by wire transfer. What is your IBAN number?
+faq_item_2_p_thunderbird_a=Unfortunately direct wire transfers from outside of the US are very expensive for us. Among other fees there is an additional fee of $15 for each incoming wire transfer, which can exceed the amount of the donation. Therefore we do not offer wire transfers as a payment method for the moment.
+faq_item_2_p_thunderbird_b=We're aware that for Europeans donating via IBAN number would be very convenient, and we're working on making that possible.
 faq_item_3_header=Why do you need my address in order to process a donation?
 faq_item_3_paragraph=We understand that your privacy is very important. We ask for a minimum amount of information required to process credit card donations, including billing addresses. This allows our payment processor to verify your identity, process your payment, and prevent fraudulent charges to your credit card. We keep your information private -- if you have questions, please refer to our <a href="https://www.mozilla.org/privacy/">Privacy Policy</a>. If you would rather not fill in your information on our online donation form, you can <a href="https://donate.mozilla.org/ways-to-give">mail a check</a> or donate directly <a href="https://donate.mozilla.org/paypal-donate">via Paypal</a>.
+faq_item_3_paragraph_thunderbird=We understand that your privacy is very important. We ask for a minimum amount of information required to process credit card donations, including billing addresses. This allows our payment processor to verify your identity, process your payment, and prevent fraudulent charges to your credit card. We keep your information private -- if you have questions, please refer to our <a href="https://www.mozilla.org/privacy/">Privacy Policy</a>. If you would rather not fill in your information on our online donation form, you can mail us a check.
 faq_item_4_header=Where do I send a check/cheque?
-faq_item_4_paragraph_a=Please send your check to:
```

Comment:
Not sure how I missed that, but this string is still used on MoFo faq and should not be removed